src/Sulu/Bundle/AdminBundle/Tests/app/config/config.yml

```diff
@@ -15,7 +15,6 @@ security:
                 check_path: sulu_admin.login_check
                 success_handler: sulu_security.authentication_handler
                 failure_handler: sulu_security.authentication_handler
-                csrf_provider: security.csrf.token_manager
```

Comment:
Will that also be removed in sulu-standard and sulu-minimal? Is the CSRF protection still working?